source/_integrations/russound_rio.markdown

```diff
@@ -14,9 +14,9 @@ ha_config_flow: true
 ha_integration_type: integration
 ---
 
-The `russound_rio` {% term integration %} allows you to control Russound devices that make use of the RIO protocol.
+The `russound_rio` {% term integration %} allows you to control Russound devices that make use of the RIO protocol. All devices that are compatible with the [Russound app](https://www.russound.com/russound-app) are compatible.
```

Comment:
Perhaps it may be more suitable to mention this link under the "Supported Devices" section. Also the list under "Supported Devices" may be outdated if you compare them.

Reply:
@silamon Ended up removing it on second thought. There is a device that is compatible with the app that isn't compatible with this implementation. The supported devices list remains the source of truth.